src/sentry/static/sentry/app/components/gridEditable/index.tsx

```diff
@@ -318,6 +320,21 @@ class GridEditable<
     );
   }
 
+  renderDownloadAllButton(canEdit: boolean) {
+    const disabled = this.props.isLoading || canEdit === false;
+    return (
+      <HeaderExportButton
+        payload={{
+          queryType: 'Discover',
+          queryInfo: this.props.location.query,
+        }}
+        disabled={disabled}
+      >
+        <InlineSvg src="icon-download" />
```

Comment:
Could use the new download icon in `app/icons/iconDownload`.

Reply:
👌